llvm/tools/llvm-readobj/GOFFDumper.cpp

```diff
@@ -0,0 +1,94 @@
+#include "ObjDumper.h"
+#include "llvm-readobj.h"
+#include "llvm/Object/GOFFObjectFile.h"
+#include "llvm/Support/ScopedPrinter.h"
+
+using namespace llvm;
+using namespace llvm::object;
+
+namespace {
+
+class GOFFDumper : public ObjDumper {
+public:
+  GOFFDumper(const GOFFObjectFile *Obj, ScopedPrinter &Writer)
+      : ObjDumper(Writer, Obj->getFileName()), Obj(Obj) {}
+
+  void printFileHeaders() override {}
```

Comment:
I don't know the GOFF format at all, but will it make sense to support this (and `printUnwindInfo` and `printStackMap`) in the future? If so, I would recommend adding a warning to these methods saying that they are unimplemented.